.travis.yml

```diff
@@ -1,106 +1,62 @@
-language: php
-
-dist: xenial
-
-services:
-  - mysql
-  - postgresql
-
-cache:
-  directories:
-    - $HOME/.composer/cache/files
+version: ~> 1.0
 
 addons:
   apt:
     packages:
       - tidy
 
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/standard.yml
+
 env:
   global:
-    - COMPOSER_ROOT_VERSION=4.x-dev
+    - COMPOSER_ROOT_VERSION="4.7.x-dev"
+    - REQUIRE_RECIPE="4.7.x-dev"
 
-matrix:
+# Using a manual matrix to exlude PHPUNIT_COVERAGE_TEST for now because it was causing some strange issues
+# e.g. https://travis-ci.com/github/silverstripe/silverstripe-framework/jobs/457096837
+jobs:
   fast_finish: true
   include:
     - php: 7.1
       env:
         - DB=PGSQL
+        - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
         - PHPCS_TEST=1
-        - PHPUNIT_TEST=framework
-        - COMPOSER_INSTALL_ARG="--prefer-lowest"
```

Comment:
Why remove the `--prefer-lowest` build? I guess we're not exposing one in the shared matrix?

Reply:
Yeah phase 2 thing, get --prefer-lowest into the shared matrix.  To many broken things to fix right now.

Reply:
Can we restore it and document it as a temporary deviation? Keen to keep this coverage present here until we get capacity to add it to the shared matrix.

Reply:
I just added it back in